src/checkers/flashchecker.py

```diff
@@ -0,0 +1,130 @@
+# Flash Checker: A checker that to see if the url is pointing to the latest Flash Player.
+#
+# The contens of the x-checker-data for the module should be .e.g:
```

Comment:
```suggestion
# The x-checker-data for the module should be, for example:
```

Reply:
I'd rather this documentation lived in the `README` instead/as well.

Reply:
Err I actually copied that format from debianrepochecker.py without noticing the typo, should I fix it there as well?

Reply:
Yes please